src/test-ids/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const Ids = require('./ids.json').split('"') // file can be generated using mkg20001/test-peer-ids.tk patched with ipfs://QmVZRWjZoqve9UKgng2gymN8a3FQUENgsQaicQ2fWs7kGx
+const Id = require('peer-id')
+const loadKey = require('libp2p-crypto').keys.supportedKeys.rsa.unmarshalRsaPrivateKey // this is needed because the keys aren't wrapped in the usual privateKey pbuf container to save space
+const base = require('base-x')(' !#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{|}~') // "compression" aka "whatever part of ascii can be put into a JSON string without needing to get escaped"-base
+const rand = (min, max) => Math.floor(Math.random() * (max - min + 1)) + min
+
+const protobuf = require('protons')
+const pbm = protobuf(require('libp2p-crypto/src/keys/keys.proto'))
+
+module.exports = (cb) => {
+  loadKey(base.decode(Ids[rand(0, Ids.length)]), (err, key) => {
+    if (err) { return cb(err) }
+    key.hash((err, digest) => {
+      if (err) { return cb(err) }
+      cb(null, new Id(digest, key, key.public))
+    })
+  })
+}
+
+module.exports.privKey = () => pbm.PublicKey.encode({
+  Type: pbm.KeyType.RSA,
+  Data: base.decode(Ids[rand(0, Ids.length)])
+}).toString('base64')
```

Comment:
Can we do all this work beforehand and have `ids.json` just be a list of base64 encoded private keys?

Reply:
We'd also not need `"whatever part of ascii can be put into a JSON string without needing to get escaped"-base` if we did this.

Reply:
It's bigger than. The comment starts with `"compression"` for a reason ;)
(I wanted to just use a binary protobuf but there are browsers and we know they like to mess with binary stuff)